Lib/Database/MySqlDriver.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * MySqlDriver class file.
+ *
+ * @package Freyja\Database
+ * @copyright 2016 SqueezyWeb
+ * @author Gianluca Merlo <gianluca@squeezyweb.com>
+ * @since 0.1.0
+ */
+
+namespace Freyja\Database;
+
+use Freyja\Exceptions\InvalidArgumentException as InvArgExcp;
+use Freyja\Database\Query;
+use mysqli;
+
+/**
+ * MySqlDriver class.
+ *
+ * @package Freyja\Database
+ * @author Gianluca Merlo <gianluca@squeezyweb.com>
+ * @since 0.1.0
+ * @version 1.0.0
+ */
+class MySqlDriver implements Driver {
+  /**
+   * Driver name.
+   *
+   * @since 1.0.0
+   * @access private
+   * @var string
+   */
+  private $name;
```

Comment:
Remove property `$name`. In order to get the name of the driver attached to a `Database` instance we should use `get_class()`.